.travis.yml

```diff
@@ -18,6 +20,9 @@ install:
   - npm --production=false install
 script:
   - npm run build
+  - rvm install "ruby-2.4.1"
+  - rvm use "ruby-2.4.1"
+  - gem install bundler -v 1.17.3
```

Comment:
We also have gem install `bundler -v '< 2'` in `scripts/setup_test_environment.sh`, do we need to install it twice?

Reply:
@stefaniacardenas does that script run on Travis builds?

Reply:
Yes, check line 18 of this file

Reply:
travis failed telling me that bundler was not installed. this is why I addded it

Reply:
I see... Maybe something changed on travis, because before it was able to install it from the script file.

Reply:
Do we need to keep it in both places then?